src/System.Drawing.Common/System.Drawing.Common.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27213.1
+# Visual Studio Version 16
```

Comment:
Was this intentional? We still support VS2017

Reply:
VS does this. Doesn't block using downlevel, it's just a track of what last edited it.